R/bind_pdf.r

```diff
@@ -0,0 +1,32 @@
+#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'. 
+#'
+bind_pdfs = function(temp_out_dir, final_out_dir, write_file_name){
```

Comment:
Please add entry in `README.md` for this function. You can place it anywhere you like. It doesn't have to be super-thorough, just something a person could visually scan and think "oh, I could use that!"